app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def new
+    @user = User.find(params[:user_id])
+  end
```

Comment:
[OPTIONAL] I don't think you need a new action in this scenario since you are not using a form to create likes.

Reply:
Okay Thanks for the feedback @fobadara